app/Http/Transformers/PostTransformer.php

```diff
@@ -35,8 +35,7 @@ public function transform(Post $post)
                 'original_image_url' => $post->url,
                 'caption' => $post->caption,
             ],
-            // @TODO: include tags
-            'tags' => $post->tagSlugs(),
+            'tagged' => $post->tagSlugs(),
```

Comment:
I think this is going to break the front-end. I already updated it to work with the new response. but should be quick enough to fix back. https://github.com/DoSomething/rogue/pull/382/files#diff-18d307dbdc33d56a9696c2923ef00b20R24

Frontend testing soon to come so we don't run into this!